index.js

```diff
@@ -152,20 +152,8 @@ const ScrollableTabView = React.createClass({
           const offsetX = e.nativeEvent.contentOffset.x;
           this._updateScrollValue(offsetX / this.state.containerWidth);
         }}
-        onMomentumScrollBegin={(e) => {
-          const offsetX = e.nativeEvent.contentOffset.x;
-          const page = Math.round(offsetX / this.state.containerWidth);
-          if (this.state.currentPage !== page) {
-            this._updateSelectedPage(page);
-          }
-        }}
-        onMomentumScrollEnd={(e) => {
-          const offsetX = e.nativeEvent.contentOffset.x;
-          const page = Math.round(offsetX / this.state.containerWidth);
-          if (this.state.currentPage !== page) {
-            this._updateSelectedPage(page);
-          }
-        }}
+        onMomentumScrollBegin={this._onMomentumScrollBeginAndEnd}
```

Comment:
I've never benchmarked it but respected people says that it is good for performance 😄